import/index_java/src/org/vufind/index/DatabaseManager.java

```diff
@@ -83,9 +83,14 @@ private void connectToDatabase()
             if (dsn.substring(0, 8).equals("mysql://")) {
                 classname = "com.mysql.jdbc.Driver";
                 prefix = "mysql";
-                // For now, disable SSL to prevent warning messages.
-                // TODO: implement SSL support (VUFIND-1395):
-                extraParams = "?useSSL=false";
+                String useSsl = ConfigManager.instance().getConfigSetting("config.ini", "Database", "use_ssl");
+                if (useSsl != null) {
+                    String verifyCert = ConfigManager.instance().getConfigSetting("config.ini", "Database", "verify_server_certificate");
```

Comment:
I'd explicitly set this to "true" or "false" to avoid issues if someone sets the value to e.g. 0 in config.ini.

Reply:
Done!